src/environment.ts

```diff
@@ -45,7 +45,8 @@ export function setEnvironment(env: InstallationDetails) {
     log(chalk.bold.green(`Now configuring the Visual Studio Build Tools..`));
   } else {
     log(chalk.bold.green(`Skipping configuration: No configuration for Python or Visual Studio Build Tools required.`));
-    return;
+    return(new Promise((resolve, reject) => resolve()))
```

Comment:
Why not [`Promise.resolve()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve)?